drivers/i2c/i2c_ll_stm32.c

```diff
@@ -23,12 +23,34 @@ static int i2c_stm32_runtime_configure(struct device *dev, u32_t config)
 	const struct i2c_stm32_config *cfg = DEV_CFG(dev);
 	struct i2c_stm32_data *data = DEV_DATA(dev);
 	I2C_TypeDef *i2c = cfg->i2c;
-	u32_t clock;
+	u32_t clock = 0;
 
 	data->dev_config = config;
 
+#if defined(CONFIG_SOC_SERIES_STM32F3X) || defined(CONFIG_SOC_SERIES_STM32F0X)
+	switch ((u32_t)cfg->i2c) {
+#ifdef CONFIG_I2C_1
+	case CONFIG_I2C_1_BASE_ADDRESS:
+		clock = LL_RCC_GetI2CClockSource(LL_RCC_I2C1_CLKSOURCE);
```

Comment:
This is wrong. I entered wrong function. I want to use LL_RCC_GetI2CClockFreq but the USE_FULL_LL_DRIVER is not defined.
@erwango which is the best way to define USE_FULL_LL_DRIVER?